Amazon_schema.txt

```diff
@@ -0,0 +1,67 @@
+create table Addresses(AddressId int primary key identity(1,1),
+StreetAddress VARCHAR(100),
+City VARCHAR(50),
+State VARCHAR(50),
+PostalCode VARCHAR(10),
+Country VARCHAR(50));
+
+
+
+
+create table Users(UserId int primary key identity(1,1),
+Name varchar(50),
+PhoneNumber nvarchar(15),
+Email varchar(254) unique,
+AddressId int not null,
+foreign key(AddressId) references Addresses(AddressId));
+
+create table Categories(CategoryId int primary key identity(1,1),
+CategoryName varchar(50) not null);
+
+
+create table Products(ProductId int primary key identity(1,1),
+productName varchar(50) not null,
+CategoryId int not null,
+Price decimal(10,2) not null,
+foreign key(CategoryId) references Categories(CategoryId));
+
+
+
+create table Orders(OrderID int primary key identity(1,1),
+UserId int not null,
+TotalAmount decimal(10,2) not null,
+OrderStatus VARCHAR(20) NOT NULL, 
+OrderDate DATETIME NOT NULL DEFAULT GETDATE(),
+foreign key(UserId) references Users(UserId));
+
+create table OrderedProducts(OrderedProductsId int primary key identity(1,1),
+ProductId int not null,
+OrderId int not null,
+Quantity int not null,
+Amount decimal(10,2) not null,
```

Comment:
amount or total amount ?

Reply:
here amount is the Price * Quantity of that particular product

Reply:
Price is not storing then ?

Reply:
price is stored in the products table